src/Libraries/CoreNodeModels/CoreNodeModels.csproj

```diff
@@ -26,17 +26,11 @@
       <!--Exclude copying the runtime dlls because we will handle it in the BinaryCompatibilityOps target -->
       <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
-	  <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
     <PackageReference Include="CoreCLR-NCalc" Version="2.2.92" />
   </ItemGroup>
   <ItemGroup>
-  <PackageReference Include="ncalc" Version="1.3.8">
-    <GeneratePathProperty>true</GeneratePathProperty>
-    <!--Exclude copying the runtime dlls because we will handle it in the BinaryCompatibilityOps target -->
-    <ExcludeAssets>runtime</ExcludeAssets>
-  </PackageReference>
```

Comment:
I think the ItemGroup with all the `References` to PresentationCore etc should be under net48

Reply:
Also I am not sure why we have WPF and Windows references in a core project. 
I hope these are just junk references that we can remove. If not we need to segregate the wpf and windows only code in the project

Reply:
not sure about the WPF parts but I don't think that has been changed in years.